recipe/meta.yaml

```diff
@@ -1,31 +1,36 @@
 {% set name = "skein" %}
-{% set version = "0.8.0" %}
+{% set version = "0.8.1" %}
+{% set pkg_loc = "py2.py3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
-  sha256: 71da4a16e2bba4d99fa90454b58e951463452596a6e6afaaf024a0d1cce87d3f
+  url: https://files.pythonhosted.org/packages/{{ pkg_loc }}/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-{{ pkg_loc }}-none-any.whl
```

Comment:
Had some issues with this part of the URL changing between releases. So factored this out into its own Jinja variable in case we need to change that again